assembl/tests/views_tests/test_graphql.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+import json
+
+import pytest
+from graphql_relay.node.node import to_global_id
+
+from assembl import models
+from assembl.graphql.schema import Schema as schema
+
+
+@pytest.fixture(scope="function")
```

Comment:
Should we assume to have more graphql tests that are not based on thematic and questions? If so, then perhaps pulling the fixture out would be useful.